src/pixi/IgePixiTexture.js

```diff
@@ -27,12 +27,15 @@ var IgePixiTexture = IgeClass.extend({
 
 				// if image is not from discord
 				options = { crossOrigin: true };
-
-				resource.add(source, `${source}?version=${version}`, options)
-					.load(function () {
-						data.entity.pixianimation._anims = {};
-						data.entity[data.cb](data.animationId);
-					});
+				
+				// check if the cell sheet url (source) is a valid url
+				if (source && source.indexOf('http') === 0) {
+					resource.add(source, `${source}?version=${version}`, options)
+						.load(function () {
+							data.entity.pixianimation._anims = {};
+							data.entity[data.cb](data.animationId);
+						});
+				}
 
```

Comment:
what changed in this PR? This doesn't look like a diff from the previous change. Was the one this morning not `IgePixiTexture.js`?

Reply:
Nope, that was in client.js, pixi loader is being used at multiple places.